src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        return "fruit,quantity" + Storage.reports.entrySet().stream()
```

Comment:
Direct access to the `Storage.reports` field violates the encapsulation principle. It's better to provide a method in the `Storage` class that returns the needed data without exposing the internal structure.